Extensions/Lighting/lightruntimeobject-pixi-renderer.js

```diff
@@ -232,7 +232,6 @@ gdjs.LightRuntimeObjectPixiRenderer.prototype.updateColor = function () {
 };
 
 gdjs.LightRuntimeObjectPixiRenderer.prototype.updateTexture = function () {
-  if (!this._light) return;
```

Comment:
This makes no sense, the function will always return early without setting the texture 😬, a minor regression.